.travis.yml

```diff
@@ -17,6 +17,9 @@ cache:
 
 install:
   - npm install -g npm@5.5.1 yarn@1.3.2
+  - if [ "$TRAVIS_OS_NAME" = "linux" ]; then
```

Comment:
So far we only build on Linux with Travis, so strictly speaking that's not necessary.